client/sandbox/react-native-expo/app/config.js

```diff
@@ -2,4 +2,5 @@ export default {
   appId: process.env.EXPO_PUBLIC_INSTANT_APP_ID,
   apiURI: 'http://localhost:8888',
   websocketURI: 'ws://localhost:8888/runtime/session',
+  verbose: true,
```

Comment:
Added the nifty `verbose` flag, so we automatically see our logs in RN. H/t @dwwoelfel